### PR TITLE
Task_5_DWH

### PR DESCRIPTION
**Pull request by MariDzin**

### CODE REVIEW

**Comment by MariDzin** [comment]
i have a lot of errors for the sql code that i am trying to fix for about few hours already, as soon as i fix ill upload it too

**Comment by MariDzin** [comment]
image
i try to fix that error

File: DWH/Task5/Script-3.sql
Comment (marimldvnv) [inline DWH/Task5/Script-3.sql:1]: All attributes of all tables in source layer should have VARCHAR datatype (we must retrieve data from source as it is and if we use some predefined datatype data may be corrupted when casting).
Please use commit after each dml operation.

**Review by marimldvnv — CHANGES_REQUESTED** [review]
Please see comments. 
Were you able to fix the issue with accessing the file?
Also PLEASE remember to assign me as a reviewer to your PR, otherwise I don't see notification that you have a HW done.

File: DWH/Task5/Script-3.sql
Comment (marimldvnv) [reply DWH/Task5/Script-3.sql:1]: ...of ALL tables. in external too :) precision is up to you, but usually it is max = 255

**Review by marimldvnv — CHANGES_REQUESTED** [review]

**Review by marimldvnv — CHANGES_REQUESTED** [review]
Can you please provide your current versions of datasets? your code is not working for me
image

image

**Comment by MariDzin** [comment]
HI, i provided updated databases in the pull request called- update databases

**Review by marimldvnv — APPROVED** [review]

**Pull request merged**